src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Contracts/Identification.cs

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#nullable disable
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Workspace.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.LanguageServices.Contracts
+{
+    /// <summary>
+    /// Parameters to pass into IdentificationRequest
+    /// </summary>
+    public class IdentificationParams
```

Comment:
Maybe DetectSqlObjectTypeParams? @kburtram what do you think? Identification seems to vague to indicate what this api is actually doing